internal/utils/crosslink_storage.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+import "sync"
+
+type CrosslinkStorage struct {
```

Comment:
What's CrosslinkStorage for in this PR? Please delete if it's out of date.

Reply:
removed